api/client_test.go

```diff
@@ -1,4 +1,4 @@
-package api
+package api_test
```

Comment:
I wanted to use a private method in this file in a black box test, so I made this file a black box test too.